### PR TITLE
Build `references.toml`

### PR DESCRIPTION
**Pull request by vladsavelyev**
Describe the reference data sources in `reference.py` as code.

Add a GitHub workflow that reads changes in `references.py`, transfers data, prepares and deploys `references.toml`.

### CODE REVIEW

File: .github/workflows/changed.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
+name: Deploy references.toml
+on:
+  push:
+#    branches:
```
Comment (lgruen) [inline .github/workflows/changed.yaml:4]: Accidentally commented out?

**Review by lgruen — APPROVED** [review]
This is awesome!

File: .github/workflows/changed.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+#    paths:
+#      - 'references.py'
+
+jobs:
+  make_matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set_matrix.outputs.matrix }}
+    env:
+      REFERENCES_PREFIX: gs://cpg-reference-test
```
Comment (lgruen) [inline .github/workflows/changed.yaml:15]: Needs to be changed to the actual dataset path?

File: .github/workflows/changed.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  transfer:
+    runs-on: ubuntu-latest
+    needs:
+      - make_matrix
+    strategy:
+      matrix: ${{ fromJson(needs.make_matrix.outputs.matrix) }}
+    env:
+      CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+      PROJECT: 'cpg-common'
+      REFERENCES_PREFIX: gs://cpg-reference-test
```
Comment (lgruen) [inline .github/workflows/changed.yaml:47]: Ditto?

File: .github/workflows/changed.yaml (hunk 1 above)
Comment (vladsavelyev) [reply .github/workflows/changed.yaml:4]: Intentionally; I needed to test the CI so I removed these limitations so it's triggered on every push. Will uncomment once it's the PR is finalised

File: .github/workflows/changed.yaml (hunk 2 above)
Comment (vladsavelyev) [reply .github/workflows/changed.yaml]: Will change that to reading secrets created by infra pulumi

File: .github/workflows/changed.yaml (hunk 3 above)
Comment (vladsavelyev) [reply .github/workflows/changed.yaml]: Same

**Pull request merged**